src/scanpy/metrics/_gearys_c.py

```diff
@@ -1,3 +1,5 @@
+"""Geary's C autocorrelation."""
```

Comment:
I didn’t change anything in the two metrics modules, just reordered and reformatted (and replaced the removed `np.float_` with `np.float64`, which in numpy 1.x are identical)